packages/contracts-bedrock/scripts/deploy/DeployOPPrestateUpdater.s.sol

```diff
@@ -17,322 +17,87 @@ import { IProtocolVersions } from "interfaces/L1/IProtocolVersions.sol";
-            addressManager: _doi.addressManagerBlueprint(),
-            proxy: _doi.proxyBlueprint(),
-            proxyAdmin: _doi.proxyAdminBlueprint(),
-            l1ChugSplashProxy: _doi.l1ChugSplashProxyBlueprint(),
-            resolvedDelegateProxy: _doi.resolvedDelegateProxyBlueprint(),
-            permissionedDisputeGame1: _doi.permissionedDisputeGame1Blueprint(),
-            permissionedDisputeGame2: _doi.permissionedDisputeGame2Blueprint(),
-            permissionlessDisputeGame1: _doi.permissionlessDisputeGame1Blueprint(),
-            permissionlessDisputeGame2: _doi.permissionlessDisputeGame2Blueprint()
-        });
-
-        OPPrestateUpdater oppu_ = deployOPPrestateUpdater(_doi.superchainConfig(), _doi.protocolVersions(), blueprints);
-        _doo.set(_doo.oppu.selector, address(oppu_));
-
-        assertValidPrestateUpdater(_doi, _doo);
-    }
-
     function deployOPPrestateUpdater(
-        ISuperchainConfig _superchainConfig,
-        IProtocolVersions _protocolVersions,
-        IOPContractsManager.Blueprints memory _blueprints
+        string memory _baseChain
     )
         public
-        returns (OPPrestateUpdater oppu_)
+        returns (OPPrestateUpdater oppu)
     {
-        vm.startBroadcast(msg.sender);
-        oppu_ = OPPrestateUpdater(
+        string memory superchainBasePath = "./lib/superchain-registry/superchain/configs/";
+        string memory superchainToml = vm.readFile(string.concat(superchainBasePath, _baseChain, "/superchain.toml"));
+
+        // Superchain shared contracts
+        ISuperchainConfig superchainConfig = ISuperchainConfig(vm.parseTomlAddress(superchainToml, ".superchain_config_addr"));
+        IProtocolVersions protocolVersions = IProtocolVersions(vm.parseTomlAddress(superchainToml, ".protocol_versions_addr"));
+        IOPContractsManager180 opContractsManager180 = IOPContractsManager180(vm.parseTomlAddress(superchainToml, ".op_contracts_manager_proxy_addr"));
+
+
+        // forgefmt: disable-start
+        IOPContractsManager.Blueprints memory blueprints;
+        blueprints.addressManager = address(0);
+        blueprints.proxy = address(0);
+        blueprints.proxyAdmin = address(0);
+        blueprints.l1ChugSplashProxy = address(0);
+        blueprints.resolvedDelegateProxy = address(0);
+
+        IOPContractsManager180.Blueprints memory blueprints180 = opContractsManager180.blueprints();
+        blueprints.permissionedDisputeGame1 = blueprints180.permissionedDisputeGame1;
+        blueprints.permissionedDisputeGame2 = blueprints180.permissionedDisputeGame2;
+
+        // TODO: checkout op-contracts/v1.8.0, get the initcode, paste it in here.
+        // bytes memory faultDisputeGameInitcode = 0x.....;
+        // (blueprints.permissionlessDisputeGame1, blueprints.permissionlessDisputeGame2) = DeployUtils.createDeterministicBlueprint(faultDisputeGameInitcode, _salt);
+        // forgefmt: disable-end
```

Comment:
Perhaps can just fetch this from the google cloud artifacts, you may know [surl](https://github.com/memester-xyz/surl) for simplifying http requests from forge

Reply:
We agreed out of band to skip surl for now since this is just a temporary script.